src/ja/server/proxy/proxy.py

```diff
@@ -63,23 +51,49 @@ def resume_job(self, uid: str) -> WorkerResponse:
         """
 
 
-class WorkerProxy(IWorkerProxy):
+class WorkerProxyBase(IWorkerProxy, ABC):
     """
-    Implementation of the proxy for the worker client used on the central server.
+    Base class for a worker proxy class that internally uses an ISSHConnection object.
     """
 
+    def __init__(self, uid: str, ssh_config: SSHConfig):
+        """!
+        @param uid: The UID of the worker represented by this proxy.
+        @param ssh_config: Config for paramiko.
+        """
+        super().__init__(ssh_config=ssh_config)
+        self._uid = uid
+
     @property
     def uid(self) -> str:
-        pass
+        return self._uid
 
     def dispatch_job(self, job: Job) -> WorkerResponse:
-        pass
+        command = StartJobCommand(job)
+        response = self._ssh_connection.send_worker_command(command)
+        return response
 
     def cancel_job(self, uid: str) -> WorkerResponse:
-        pass
+        command = CancelJobCommand(uid)
+        response = self._ssh_connection.send_worker_command(command)
+        return response
 
     def pause_job(self, uid: str) -> WorkerResponse:
-        pass
+        command = PauseJobCommand(uid)
+        response = self._ssh_connection.send_worker_command(command)
+        return response
 
     def resume_job(self, uid: str) -> WorkerResponse:
-        pass
+        command = ResumeJobCommand(uid)
+        response = self._ssh_connection.send_worker_command(command)
+        return response
+
+
+class WorkerProxy(WorkerProxyBase):
+    """
+    Implementation of the proxy for the worker client used on the central server.
+    """
+
+    def _get_ssh_connection(self, ssh_config: SSHConfig) -> ISSHConnection:
+        raise NotImplementedError("Remote path not defined")
```

Comment:
Wouldn't it make sense to put the remote in `PATH` or run it with `python3 -m ...` as I suggested on the other PR? This way you don't need to hardcode the path to it.